layouts/partials/tw/modal-ribbon.html

```diff
@@ -0,0 +1,166 @@
+{{ $dot := . }}
+{{ $idEmail := partial "helper/new-id" $dot | printf "modal-newsletter-%s" }}
+{{ $imagename := .filename }}
+{{ $actionColor := .actionColor | default "#FFCB05" }}
+
+
+<div
+  class="fixed inset-0 z-20"
+  data-ga-category="modal:newsletter:multi"
+  x-data="modal"
+  x-cloak
+  x-show="isOpen"
+  x-trap.noscroll.inert="isOpen"
+  @modal:newsletters:open.window="isOpen = true"
+  {{ if .page.Param `modal-exclude` | not }}
+    @scroll.window.once.passive="trigger()"
+  {{ end }}
+  @click="isOpen = false"
+  role="dialog"
+  aria-modal="true"
+  aria-label="Newsletter sign up"
+>
+  <div class="absolute inset-0 bg-s-4 opacity-75"></div>
+  <div
+    class="absolute inset-0 backdrop-blur-sm backdrop-filter"
+    x-transition.opacity
+  ></div>
+  <div
+    class="absolute inset-0 flex items-center overflow-y-auto"
+    x-show="isOpen"
+    x-transition
+  >
+    <div class="w-full border-b-2 border-t-2 bg-[{{ $actionColor }}] font-sans">
```

Comment:
If you don't say exactly bg-[whatever] in the HTML, the CSS purge script can accidentally purge this class out. Usually, I give up and set a style. Another solution is to do `style="--bg-color: {{ $actionColor }}" class="bg-[--bg-color]"`.

Reply:
Thanks that worked!